### PR TITLE
[RaftStore] Add keys split checker

### PR DESCRIPTION
**Pull request by cfzjywxk**
#300 
Add keys split checker and related config fields

### CODE REVIEW

**Comment by cfzjywxk** [comment]
@coocood @bobotu  PTAL

File: tikv/raftstore/worker.go [hunk 1]
```diff
@@ -295,25 +305,38 @@ func exceedEndKey(current, endKey []byte) bool {
...
+/// SplitCheck gets the split keys by scanning the range.
 func (r *splitCheckHandler) splitCheck(startKey, endKey []byte, reader *dbreader.DBReader) [][]byte {
 	ite := reader.GetIter()
 	splitKeys := r.tryTableSplit(startKey, endKey, ite)
 	if len(splitKeys) > 0 {
 		return splitKeys
 	}
-	checker := newSizeSplitChecker(r.config.regionMaxSize, r.config.regionSplitSize, r.config.batchSplitLimit)
+	r.newCheckers()
+	abort := false
```
Comment (coocood) [inline tikv/raftstore/worker.go]: The name `abort` is not appropriate.

File: tikv/raftstore/worker.go [hunk 2]
```diff
@@ -295,25 +305,38 @@ func exceedEndKey(current, endKey []byte) bool {
...
 	for ite.Seek(startKey); ite.Valid(); ite.Next() {
 		item := ite.Item()
 		key := item.Key()
 		if exceedEndKey(key, endKey) {
 			break
 		}
-		if checker.onKv(key, uint64(item.ValueSize())) {
+		for _, checker := range r.checkers {
```
Comment (coocood) [inline tikv/raftstore/worker.go]: If we extract the for loop into a function, we don't need to define an `abort` variable before the outer loop.

**Comment by coocood** [comment]
LGTM

**Pull request merged**